.github/workflows/release.yml

```diff
@@ -58,13 +58,19 @@ jobs:
       - name: Check out code into the Go module directory
         uses: actions/checkout@v2
 
+      - name: Verify go modules
+        run: go mod verify
+
       - name: Check formatting
         run: |
-          chmod +x $GITHUB_WORKSPACE/scripts/gofmtcheck.sh
-          $GITHUB_WORKSPACE/scripts/gofmtcheck.sh
+            chmod +x $GITHUB_WORKSPACE/scripts/gofmtcheck.sh
```

Comment:
Was this intentionally indented a couple extra spaces? The below `run:` value for `Linting` is not indented the same amount. It obviously does not matter, but just calling it out.

Reply:
Fixed!